package.json

```diff
@@ -82,5 +82,6 @@
     },
     "resolutions": {
         "merkle-patricia-tree": "^2.3.2"
-    }
+    },
+    "version": "0.0.0"
```

Comment:
I have no idea where these changes and the `v0.0.0` commit come from. Please ignore. :D